group_project_v2/mixins.py

```diff
@@ -222,3 +230,26 @@ def studio_view(self, context):  # pylint: disable=unused-argument, no-self-use
         fragment = Fragment()
         fragment.add_content(NO_EDITABLE_SETTINGS)
         return fragment
+
+
+class DashboardMixin(object):
+    def dashboard_view(self, context):
```

Comment:
@Kelketek I've tried using abstractmethods and refrained from that for some reason I can't recall now. Most likely it had something to do with metaclasses and XBlock stuff (like using ABCMeta overwritten the metaclass set in XBlock and made XBlock totally broken)